pytorch_lightning/callbacks/progress.py

```diff
@@ -71,14 +71,14 @@ class ProgressBarBase(Callback):
         class LitProgressBar(ProgressBarBase):
 
             def __init__(self):
-                super().__init__()  # don't forget this :)
```

Comment:
you removed the little bit of happiness that I added 😄

Reply:
Expecto patronum!

https://github.com/PyTorchLightning/pytorch-lightning/pull/8836/commits/0868e15a98d0ee6f87f7162d4bf05dccd02d1814